config/samples/watcher_v1beta1_watcher.yaml

```diff
@@ -3,10 +3,10 @@ kind: Watcher
 metadata:
   labels:
     app.kubernetes.io/name: watcher
-    app.kubernetes.io/instance: watcher-sample
+    app.kubernetes.io/instance: watcher
```

Comment:
so just so your aware, there are pros and cons to this renaming.

by using the name that will be used by the OpenStack operator this will look more like
a real deployment after the integration is done.

however it means that this sample should not be used with our kuttle tests as once the meta operator knows about watcher it will delete any cr with this name that it did not create when watcher is disabled in the OpenStack control plane cr.

that is why the kuttl tests should not use the same name as the metaoper will use.


that is fine it just means we should not use this in the kuttl tests.


we should use this in env test however.

can you port https://github.com/openstack-k8s-operators/nova-operator/blob/main/test/functional/sample_test.go in this PR as well so that our functional tests validate that the samples are correct.

Reply:
We are using watcher-kuttl in kuttl tests to make obvious where it is coming from and avoid mixing with others.

Thanks for the hint about sample_test.go, i will do it.